deploy/helm-chart/values.yaml

```diff
@@ -58,6 +58,17 @@ nginx:
     tag: edge
     pullPolicy: Always
 
+  ## The configuration for the data plane that is contained in the NginxProxy resource.
+  config: {}
+  #   http:
+  #     telemetry:
+  #       tracing:
+  #         enable: true # default false
+  #         endpoint: simplest-collector.default.svc.cluster.local:4317 # required
+  #         interval: "5s" # default
```

Comment:
I don't think we need the `default` comment strings. The values we have in here are implied to be the defaults.